LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
The license doesn't show up on readthedocs. I think you need to change 'LICENSE.md' into 'LICENSE' (also in license_link.md), and add the "#" so that sphinx can find it. It works on [mine](https://collaborate-jianghan.readthedocs.io/en/latest/).

Reply:
Do you mean that on [https://collaborate.readthedocs.io/en/latest/?](https://collaborate.readthedocs.io/en/latest/?) there is no License file linked? Thats because my PR isn't merged yet into main. Once it is merged, I can update the website...

Reply:
Oh ok cool! I thought you published your branch